packages/utilities/src/index.ts

```diff
@@ -8,6 +8,7 @@ export * from './DelayedRender';
 export * from './EventGroup';
 export * from './FabricPerformance';
 export * from './GlobalSettings';
+export * from './IClassNames';
```

Comment:
add a change file and i''ll merge this fast :)

Reply:
done